### PR TITLE
Feat v2/add nuxt i18n

### LINKED ISSUE #83: [V2][Front] Implement internationalization i18n
## Description
We need to implement internationalization using [i18n]( https://vue-i18n.intlify.dev/). A merge request should have already been created.

## Priority

### PR DESCRIPTION
**Pull request by Ngob**

### CODE REVIEW

File: apps/front/src/pages/users/index.vue [hunk 1]
```diff
@@ -49,6 +55,7 @@ const {
   pending: usersPending,
   refresh: usersRefresh,
 } = await useListUsers();
+//usersRefresh();
```
Comment (Ngob) [inline apps/front/src/pages/users/index.vue]: Unused code

File: apps/front/src/plugins/primevue.ts [hunk 2]
```diff
@@ -3,12 +3,15 @@ import Button from "primevue/button";
...
+import InputText from "primevue/inputtext";
 
 export default defineNuxtPlugin((nuxtApp) => {
   nuxtApp.vueApp.use(PrimeVue, { ripple: true });
   nuxtApp.vueApp.component("Button", Button);
   nuxtApp.vueApp.component("TieredMenu", TieredMenu);
+  nuxtApp.vueApp.component("InputText", InputText);
   nuxtApp.vueApp.component("DataTable", DataTable);
   nuxtApp.vueApp.component("Column", Column);
+
```
Comment (Ngob) [inline apps/front/src/plugins/primevue.ts:15]: unused line

File: apps/front/src/locales/fr.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    welcome: "Bienvenu",
+  },
+  user: {
+    list: {
+      title: "Utilisateurs",
+      edit: "Editer",
+      delete: "Supprimer",
+    },
+    pending:
+      "Pas d'utilisateurs | Chargement de l'utilisateur | Chargement des utilisateurs",
```
Comment (Ngob) [inline apps/front/src/locales/fr.ts:23]: wrong trad

File: apps/front/src/locales/en.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    welcome: "Welcome",
+  },
+  user: {
+    list: {
+      title: "Users",
+      edit: "Edit",
+      delete: "Delete",
+    },
+    pending:
+      "Pas d'utilisateurs | Chargement de l'utilisateur | Chargement des utilisateurs",
```
Comment (Ngob) [inline apps/front/src/locales/en.ts]: Wrong traduction

File: apps/front/src/components/layout/AppHeader.vue [hunk 5]
```diff
@@ -1,10 +1,8 @@
 <template>
   <div class="grid">
-    <div class="col-2 col-offset-8">
-      <NuxtLink to="/auth/register">Register</NuxtLink>
-    </div>
+    <div class="col-2 col-offset-8"></div>
     <div class="col-2">
-      <p v-if="username">Welcome {{ username }}</p>
+      <p v-if="username">{{ $t("header.welcome") }} {{ username }}</p>
```
Comment (Ngob) [inline apps/front/src/components/layout/AppHeader.vue]: Add username as variable

**Comment by nguyenk** [comment]
use filepath for key prefix, use underscore for last part, ex:
 * `pages.user.edit.title`
 * `components.table.paginatedTable.sort_asc_tooltip`
 
 Check how to plug backend validation

**Review by homersimpsons — APPROVED** [review]
about `snake_case` vs `camelCase` I have no strong opinion against this as long as it is consistent

**Pull request merged**